Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -109,17 +111,17 @@ What does it look like?
 
 Below is an example of a frontend user with its stored password hash. Since TYPO3 can handle multiple
 different hash mechanisms in parallel, each hash is prefixed with a unique string that identifies the
-used hash algorithm. In this case it is `$argon2i` which denotes the Argon2i hash algorithm:
+used hash algorithm. In this case it is `$argon2id` which denotes the Argon2id hash algorithm:
 
 .. code-block:: none
    :caption: Data of a frontend user in the database
 
    MariaDB [cms]> SELECT uid,username,password FROM fe_users WHERE uid=2;
-   +-----+----------+---------------------------------------------------------------------------------------------------+
-   | uid | username | password                                                                                          |
-   +-----+----------+---------------------------------------------------------------------------------------------------+
-   |   2 | someuser | $argon2i$v=19$m=16384,t=16,p=2$WFdVRjdqVy9TbVJPajNqcA$vMDP/TBSR0MSA6yalyMpBmFRbCD8UR4bbHZma59yNjQ |
-   +-----+----------+---------------------------------------------------------------------------------------------------+
+   +-----+----------+----------------------------------------------------------------------------------------------------+
```

Comment:
The table will not show as a table within the code-block. Is this intended?

Reply:
Yup it is, as this is the output of the result of the select statement (in MySQL/MariaDB).